site/ja/tutorials/quickstart/beginner.ipynb

```diff
@@ -180,11 +180,7 @@
         "  tf.keras.layers.Dense(128, activation='relu'),\n",
         "  tf.keras.layers.Dropout(0.2),\n",
         "  tf.keras.layers.Dense(10, activation='softmax')\n",
```

Comment:
[Suggestion]
```suggestion
        "  tf.keras.layers.Dense(10)\n",
```

Reply:
Thank you for your review! I'll fix this issue by applying the `nb_code_sync.py` tool.